### PR TITLE
refactor: ZENKO-1132 Healthcheck Arsenal errors

### PR DESCRIPTION
**Pull request by bennettbuchanan**

### CODE REVIEW

File: lib/api/Healthcheck.js [hunk 1]
```diff
@@ -74,16 +75,16 @@ class Healthcheck {
      * @param {function} cb - callback(error, data)
```
Comment (jonathan-gramain) [inline lib/api/Healthcheck.js:76]: can you update the JSDoc with `log` argument?

**Comment by bennettbuchanan** [comment]
@bert-e wait

**Review by philipyoo — APPROVED** [review]
lgtm, I have a weird error propagation in the api so that the server request logger would receive and log the error on a request basis.

Maybe a future change to make is to send the request logger down to the routes

**Review by jonathan-gramain — APPROVED** [review]

**Comment by rahulreddy** [comment]
Let's not include refactors in the upcoming build. It's meant for bug fixes only. I don't want to risk regression.

**Comment by bennettbuchanan** [comment]
@rahulreddy I agree—I figured I would open this while we were working in that part of the codebase. Please do not remove the wait instruction to bert-e. 🙏

**Comment by bennettbuchanan** [comment]
@bert-e reset

**Comment by bennettbuchanan** [comment]
@bert-e reset

**Comment by bennettbuchanan** [comment]
@bert-e help

**Comment by bennettbuchanan** [comment]
Closing in favor of PR using branching naming scheme for bert-e: https://github.com/scality/backbeat/pull/502

**Pull request closed without merging**